All Core Devs Meetings/meeting 94.md

```diff
@@ -0,0 +1,437 @@
+# All Core Devs Meeting 94 Notes
+### Meeting Date/Time: Friday, Aug 21 2020, 14:00 UTC
+### Meeting Duration: 1:32 hrs
+### [GitHub Agenda](https://github.com/ethereum/pm/issues/200)
+### [Audio/Video of the meeting](https://www.youtube.com/watch?v=q6bIeSu7r9o&t=2835s)
+### Moderator: Hudson Jameson
+### Notes: Alita Moore
+  
+---
+  
+# Summary
+  
+## EIP Status
+EIP | Status
+  
+  
+## Decisions Made
+  
+Decision Item | Description
+--|--
+**94.1.1**: Discuss the possibility of launching a Prog POW on a test net next week in line with the Ben's compromise
+**94.1.2**: Move further discussion on precompile to ETH Magicians forum [here](https://ethereum-magicians.org/t/evm384-feedback-and-discussion/4533)
+  
+  
+  
+## Actions Required
+  
+Action Item | Description
+--|--
+**94.2.1**: Decide, by next meeting, whether to go with Axic's and update the sources that say we're going with EIP 2537
```

Comment:
**94.2.1**| Decide, by next meeting, whether to go with Axic's and update the sources that say we're going with EIP 2537
**93.2.2**| Vitalik to start coordinating a writeup on the impact of the gas limit increase (what will it break)
**93.2.3**| revisit the discussion on EIP 384